README.md

```diff
@@ -1,13 +1,13 @@
 # Overview
-Library for common Synopsys Jenkins plugin logic
+Library for common Blackduck Jenkins plugin logic
```

Comment:
Here, as company/brand name, it should be Black Duck. Only in case of alias, would it be blackduck.